tools/run_tests/performance/loadtest_template.py

```diff
@@ -0,0 +1,190 @@
+        'apiVersion': 'e2etest.grpc.io/v1',
+        'kind': 'LoadTest',
+        'metadata': metadata,
+    }
+    for input_file_name in input_file_names:
+        with open(input_file_name) as f:
+            input_config = yaml.safe_load(f.read())
+
+            if input_config.get('apiVersion') != template['apiVersion']:
+                raise ValueError('Unexpected api version in file {}: {}'.format(
+                    input_file_name, input_config.get('apiVersion')))
+            if input_config.get('kind') != template['kind']:
+                raise ValueError('Unexpected kind in file {}: {}'.format(
+                    input_file_name, input_config.get('kind')))
+
+            for client in input_config['spec']['clients']:
+                if client['language'] in client_languages:
+                    continue
+                clients.append(client)
+                client_languages.add(client['language'])
+
+            for server in input_config['spec']['servers']:
+                if server['language'] in server_languages:
+                    continue
+                servers.append(server)
+                server_languages.add(server['language'])
+
+            input_spec = input_config['spec']
+            del input_spec['clients']
+            del input_spec['servers']
+            del input_spec['scenariosJSON']
+            spec.update(input_config['spec'])
+
+    clients.sort(key=lambda x: x['language'])
+    servers.sort(key=lambda x: x['language'])
+
+    spec.update({
+        'clients': clients,
+        'servers': servers,
+    })
+
+    template['spec'] = spec
+
+    loadtest_set_keys(template, keys)
+
+    return template
+
+
+def main() -> None:
+    argp = argparse.ArgumentParser(
+        description='Creates a load test config generator template.')
+    argp.add_argument('-i',
+                      '--inputs',
+                      action='extend',
+                      nargs='*',
+                      type=str,
+                      required=True,
+                      help='Input files.')
+    argp.add_argument('-o',
+                      '--output',
+                      type=str,
+                      help='Output file. Outputs to stdout if not set.')
+    argp.add_argument('-k',
+                      '--keys',
+                      action='extend',
+                      nargs='*',
+                      default=[],
+                      type=str,
+                      help='Value of keys to insert, in the form key=value.')
+    argp.add_argument('-n',
+                      '--name',
+                      default='',
+                      type=str,
+                      help='Name to insert.')
+    argp.add_argument('-a',
+                      '--annotations',
+                      action='extend',
+                      nargs='+',
+                      default=[],
+                      type=str,
+                      help='Annotations to insert, in the form key=value.')
```

Comment:
help string should mention that these will become metadata.annotations in the final template.

Reply:
DOne.